src/wp-includes/class-wp-theme-json.php

```diff
@@ -2736,6 +2765,20 @@ protected static function remove_insecure_settings( $input ) {
 				}
 			}
 		}
+
+		// Ensure indirect properties not handled by `compute_style_properties` are allowed.
```

Comment:
Nitpick: Technically, `compute_style_properties` is not used here, so this comment can be a bit misleading/confusing. Maybe we could refer to `PRESETS_METADATA`?

```suggestion
		// Ensure indirect properties not included in any `PRESETS_METADATA` value are allowed.
```

Reply:
Oh, good point, thanks for the note — I've committed that change 🙂